lib/ansible/modules/notification/ms_teams.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2019, Christian Kaiser <c.kaiser () also.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: ms_teams
+version_added: 2.9
+short_description: Send messages to a Microsoft Teams channel
+author: Christian Kaiser (@ckaiser79)
+description:
+  - Setup a webhook in Microsoft Teams. Using this webhook you can send html
+    notifications.
+  - The module enabley you to write longer text in plain text or html,
+    instead of sending direct json to the endpoint.
+options:
+  body:
+    required: true
+    description:
+      - The content of the message to be published.
+  subject:
+    required: true
+    description:
+      - The title of the message.
+  webhook:
+    required: true
+    description:
+      - The ms teams configured webhook
+  color:
+    description:
+      - Which color should be shown on the card. HTML code without a leading '#'
+    default: ffffcc
+notes:
+  - "See: https://docs.microsoft.com/de-de/outlook/actionable-messages/message-card-reference"
```

Comment:
Would be more helpful to link to english version https://docs.microsoft.com/en-gb/outlook/actionable-messages/message-card-reference